Misc/NEWS.d/next/Library/2024-09-23-18-18-23.gh-issue-124309.iFcarA.rst

```diff
@@ -0,0 +1,2 @@
+Modernize the :func:`!asyncio.staggered.staggered_task` function to now
```

Comment:
You should mention that this was also a bugfix as well as a refactor

Reply:
I think I'm just going to mention the bugfix only, a refactor is not a user-facing change.